geth/testing/testing.go

```diff
@@ -89,6 +92,30 @@ func (s *BaseTestSuite) StartTestNode(networkID int, upstream bool) {
 	require.True(s.NodeManager.IsNodeRunning())
 }
 
+// EnsureNodeSync ensures that synchronization of the node is done once and that it
+// is done properly else, the call will fail.
+func (s *BaseTestSuite) EnsureNodeSync() {
```

Comment:
This method is duplicated in this PR.

Reply:
Yes it is but due to `api.BackendTestSuite` implementing it's own TestSuite directly by embedding `testify.Suite` we have to duplicate it to `BackendTestSuite` to be able to use it. `BackendTestSuite` can not share `testing.BaseTestSuite` due its desire to override some functions for its use, i believe.